packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -103,6 +103,8 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
     } = this.props;
 
     const { menuProps } = this.state;
+    const isExpanded = !!menuProps && !!!menuProps.hidden;
```

Comment:
>const isExpanded = !!menuProps && !!!menuProps.hidden; [](start = 4, length = 54)

Couldn't a menu be styled as "expanded" while it is hidden?

Reply:
The hidden prop in IContextualMenuProps is used for perf issues. I don't think that the user needs to navigate through the hidden menu.
If the isExpanded bool is true, then the base button will have a different gray color as if the menu is opened.